site/public/js/server.js

```diff
@@ -386,12 +386,12 @@ function deletePlagiarismResultAndConfigForm(form_action, gradeable_title) {
 function addMorePriorTermGradeable(prior_term_gradeables) {
     var form = $("#save-configuration-form");
     var prior_term_gradeables_number = $('[name="prior_term_gradeables_number"]', form).val();
-    var to_append = '<br /><select name="prev_sem_'+ prior_term_gradeables_number +'"><option value="">None</option>';
+    var to_append = '<select name="prev_sem_'+ prior_term_gradeables_number +'"><option value="">None</option>';
```

Comment:
You can make this `const`, since JS is a little weird when it comes to variable scoping try to use `let` or `const` to avoid JS weird behavior

Reply:
I changed it to `let` (as its value is getting updated) and for in general, we do have all other variables using var instead of let or const... can we make an issue to update such files one-by-one with the latest syntax?